migrations/20230201519590_add_flow_state_table.up.sql

```diff
@@ -0,0 +1,20 @@
+-- see: https://stackoverflow.com/questions/7624919/check-if-a-user-defined-type-already-exists-in-postgresql/48382296#48382296
+do $$ begin
+    create type code_challenge_method as enum('s256', 'plain');
+exception
+    when duplicate_object then null;
+end $$;
+create table if not exists {{ index .Options "Namespace" }}.flow_state(
+       id uuid primary key,
+       user_id uuid null,
+       auth_code text unique not null,
```

Comment:
@hf iirc you wanted to remove the unique constraint here - is this still a concern?